content/actions/reference/workflow-commands-for-github-actions.md

```diff
@@ -158,6 +158,25 @@ Creates an error message and prints the message to the log. You can optionally p
 echo "::error file=app.js,line=10,col=15::Something went wrong"
 ```
 
+### Grouping log lines
+
+```
+::group::{title}
+::endgroup::
+```
+
+Emitting a `group` with a title will instruct the logs to create a collapsible region up to the next `endgroup` command.
```

Comment:
It looks like the description from https://github.com/actions/toolkit/blob/main/docs/commands.md#group-and-ungroup-log-lines. Maybe we can expand that description just a bit.

```suggestion
Creates a collapsible group in the log. To create a group, use the `group` command and specify a `title`. Anything you print to the log between the `group` and `endgroup` commands is nested inside a collapsible group in the log.
```

Reply:
I like this change. Maybe change the last bit to 

> nested inside an expandable entry in the log.

My idea is to use something less repetitive here and also to provide a slightly different description of what this actually is so that the user has a better chance of understanding it. Or would this make it more confusing than helpful in the end, if it switches between "collapsible group" and "expandable entry"?

Reply:
Actually, "collapsible group" and "expandable group" are different. The former starts expanded, while the latter can be collapsed. The default state in the Web UI is collapsed, which would make it an expandable group (if you watch the log live, then there is actually a moment where it's expanded, but it gets collapsed a bit later)

Reply:
This suggestion sounds great and I agree it's less repetitive and more accurate with expandable rather than collapsible! I'll go ahead and commit this!